scripts/utility.py

```diff
@@ -1014,7 +1014,7 @@ def event_text_adjust(Cat,
 
     if murder_reveal and victim:
         victim_cat = Cat.fetch_cat(victim)
-        text = text.replace("mur_c", str(victim_cat.name))
+        cat_dict["mur_c"] = (str(victim_cat.name), choice(victim_cat.pronouns))
```

Comment:
why `choice(victim_cat.pronouns)` ? shouldn't there be one, but I don't know how the pronouns work

Reply:
There can be multiple!